client/keys/add.go

```diff
@@ -95,7 +95,7 @@ func runAddCmd(cmd *cobra.Command, args []string) error {
 
 		// ask for a password when generating a local key
 		if !viper.GetBool(client.FlagUseLedger) {
-			pass, err = client.GetCheckPassword(
+			encrypPassword, err = client.GetCheckPassword(
 				"> Enter a passphrase for your key:",
```

Comment:
Can we keep the additional explanatory text ("encrypt your key to disk"); I think it's useful.

Reply:
yeah sure

Reply:
done